dagmc_stats.py

```diff
@@ -151,3 +152,47 @@ def get_surfaces_per_volume(my_core, entityset_ranges):
     for volumeset in entityset_ranges['Volumes']:
         s_p_v[volumeset] = my_core.get_child_meshsets(volumeset).size()
     return s_p_v
+
+
+def get_triangle_aspect_ratio(my_core, meshset):
+    """
+    Gets the triangle aspect ratio (according to the equation: (abc)/(8(s-a)(s-b)(s-c)), where s = .5(a+b+c).)
+    
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    entityset_ranges : a dictionary with one entry for each entityset type, and the value is the range of entities that corrospond to 
+                        each type
+    meshset : a meshset containing a certain part of the mesh
+    
+    outputs
+    -------
+    t_a_r : (list) the triangle aspect ratios for all triangles in the meshset
+    """
+    
+    t_a_r = []
+    indexes = [1,2,0]
+    
+    tris = my_core.get_entities_by_type(meshset, types.MBTRI)
+    
+    for triangle in tris:
+        side_lengths = []
+        s = 0
+        coord_list = []
+        
+        verts = list(my_core.get_adjacencies(triangle, 0))
+        
+        for side in range(3):
+            coords = my_core.get_coords(verts[side])
+            coord_list.append(coords)
+        
+        for length in range(3):    
+            side_lengths.append(np.linalg.norm(coord_list[length]-coord_list[indexes[length]]))
+            
+        s = .5*(sum(side_lengths))
+        top = np.prod(side_lengths)
+        bottom = 8*(s-side_lengths[0])*(s-side_lengths[1])*(s-side_lengths[2])
```

Comment:
```suggestion
        bottom = 8*np.prod(s-side_lengths)
```
`numpy` will do this scalar - vector correctly